spec/mailers/rrd_completed_mailer_spec.rb

```diff
@@ -42,8 +42,14 @@
       submission
     end
 
-    it 'has the expected subject' do
-      expect(email.subject).to include 'RRD claim - Processed'
+    it 'has the expected subject in production' do
+      Settings.vsp_environment = 'production'
```

Comment:
Just a left-field observation: I'm reminded of that article you shared: because this is environmental, it introduces potential memory leak across randomized tests. While it probably isn't a huge deal here, is there a chance it could affect other things?

Reply:
`Settings.vsp_environment` is `nil` in the test environment. I went ahead and restore it at the end of the test just in case.